sdk/keyvault/azure-keyvault-keys/tests/test_examples_keys_async.py

```diff
@@ -19,7 +18,7 @@ def test_create_key_client():
     # [START create_key_client]
 
     from azure.identity.aio import DefaultAzureCredential
-    from azure.security.keyvault.aio import KeyClient
+    from azure.keyvault.keys.aio import KeyClient
 
     # Create a KeyClient using default Azure credentials
     credential = DefaultAzureCredential()
```

Comment:
I think this is existing but needs to be updated to `DefaultAzureCredential()` ?

Reply:
Nevermind, I guess this was an intentional update.

Reply:
I don't follow, this is already using `DefaultAzureCredential`?

Reply:
Oh, I meant `AsyncDefaultAzureCredential`

Reply:
Removed that yesterday with #6063 😎